src/CoreBundle/Controller/ExceptionController.php

```diff
@@ -86,6 +86,10 @@ public function showAction(Request $request, FlattenException $exception, DebugL
         $request->attributes->set('pagedef', $exceptionPageDef);
         $request->query->set('pagedef', $exceptionPageDef);
 
+        // Do not execute (original) module_fnc for error pages
```

Comment:
Further testing: it is at least necessary.
Otherwise it will execute - at least - any given module function on the access denied page.

Reply:
As discussed offline: Please check if the behavior without this change leads to actual problems. Only change if it is really needed, and event then we need to think about negative consequences.

Reply:
It does currently not lead to directly visible problems.
(There is no warning about a non-existent modul function as it was the case in 6.1.)

The execution of the still present function is either halted because it does not exist in the extranet module. Or for functions which exist there (like the ChangeUserEmail with which I tested) it is stopped because no valid token is present.
(In \ChameleonSystem\CoreBundle\Controller\ChameleonController::ExecuteModuleMethod() by isModuleMethodCallAllowed())

Both checks however are not explicitly meant for this case...